README.md

```diff
@@ -47,11 +47,14 @@ zAppBuild supports a number of build scenarios:
 * **Merge Build** - Build only changed programs which will be merged back into the mainBuildBranch by using a triple-dot git diff. 
 * **Scan Source** - Skip the actual building and only scan source files to store dependency data in collection (migration scenario).
 * **Scan Source + Outputs** - Skip the actual building and only scan source files and existing load modules to dependency data in source and output collection (migration scenario with static linkage scenarios).
+* **Build Preview** - Supplemental build option. Process all phases of the supplied build option, but will not execute the commands and not create a build result.
```

Comment:
Is the build result created or not? From the code, I see it's actually created with STATUS set to 4.. This line in the README says it's not created. Can you fix that please?

Reply:
Readme is updated. Can you please have another look?